markdown/dev/reference/api/config/name/en.md

```diff
@@ -2,8 +2,17 @@
 title: name
 ---
 
+The `name` key in the pattern configuration holds the name of your design.
+
+## Structure
+
+The value should hold a string that is also a [valid NPM package 
+name](https://github.com/npm/validate-npm-package-name).
+
+## Example
+
 ```js
 name: "sorcha"
 ```
 
-A string with the name of your pattern.
+In this example, the pattern is named **Sorcha**.
```

Comment:
nit/question: does the name actually get capitalized for you?

Reply:
No, this is misleading. It probably was just muscle memory.

Reply:
Fixed